docs/guide/writing-tasks.md

```diff
@@ -53,6 +53,10 @@ task:
 
     **Note:** instructions within a task can only be named via a prefix (e.g. `test_script`).
 
+!!! tip "Task Creation for Beginners"
+    If you are just getting started and prefer a more simple way of creating tasks, the
```

Comment:
"simple" here is questionable. IMO copy and pasting from examples is the simplest way.

```suggestion
    If you are just getting started and prefer a more visual way of creating tasks, there
```

Reply:
'the' should not be 'there' in this case.

Reply:
change it to work with the next line "there is a third-party"

Reply:
Ah